src/app/modules/qsa/components/modals/calculation/calculation.modal.html

```diff
@@ -0,0 +1,33 @@
+<div class="modal-header">
+    <h5 class="modal-title">Calculated using the following formula</h5>
+    <button (click)="activeModal.dismiss()"
+            aria-label="Close"
+            class="close"
+            type="button">
+        <span aria-hidden="true">&times;</span>
+    </button>
+</div>
+<div class="modal-body">
+    <div class="lg-rounded-border">
+        <h3 class="formula-header">General formula</h3>
```

Comment:
I'd add a small margin between the elements: `m-2` or at least to the top and bottom: `mt-2 mb-2`